### ISSUE TITLE
running `pod install` seems like updates all pods

### ISSUE THREAD

**Issue by nebiros** [leavez/cocoapods-binary#35]
I that the default behavior?, when I try this plugin, I run `bundle exec pod install` after setting up my `Podfile`, then it start "installing" newer versions of the pods, seems like isn't using the `Podfile.lock` reference?

From the lock file, for example:

`- SwiftMessages (6.0.1):`

Then when I run `bundle exec pod install` after setting up my `Podfile`:

```
18:26 $ bundle exec pod install
🚀  Prebuild frameworks
Analyzing dependencies
Pre-downloading: `DVR` from `https://github.com/nebiros/DVR`
Pre-downloading: `EBCardCollectionViewLayout` from `https://github.com/betzerra/EBCardCollectionViewLayout`
Downloading dependencies
…
Installing SwiftMessages (6.0.2)
```

**Comment by hardworker** [comment]
Here is #41

**Comment by leavez** [comment]
fixed in 0.4.2
